### PR TITLE
Bugfix/divide by zero

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

Include `nullif` to all division fields to prevent divide by zero 
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue https://github.com/fivetran/dbt_pendo/issues/15
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/ci/sample.profiles.yml [hunk 1]
```diff
@@ -42,12 +42,12 @@ integration_tests:
...
       database: "{{ env_var('CI_SNOWFLAKE_DBT_DATABASE') }}"
       warehouse: "{{ env_var('CI_SNOWFLAKE_DBT_WAREHOUSE') }}"
-      schema: pendo_integration_tests
+      schema: pendo_integration_tests_2
       threads: 8
     spark:
       type: spark
       method: http
-      schema: pendo_integration_tests_new
+      schema: pendo_integration_tests_2_new
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/ci/sample.profiles.yml:50]: Can we make sure the schema names are consistent for all warehouses

File: integration_tests/dbt_project.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ dispatch:
 
 vars:
   pendo_source:
+    pendo_schema: pendo_integration_tests_2
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:15]: This should be consistent with the schema used in all CircleCi warehouse configs

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks @fivetran-reneeli! A few minor suggestions to update the integration tests schema being used. Additionally, we don't want to include the `.gspickle` and `.msgpack` files in our docs folder. This can cause errors when deploying the pages. Can you remove those before merging. In all the following should be done before merging:
- [x] Remove the `docs/graph.gpickle` file
- [ ] Remove the `docs/partial_parse.msgpack` file
- [ ] Address the comments below

**Comment by fivetran-reneeli** [comment]
Hey @fivetran-joemarkiewicz  thanks for reviewing! Didn't notice that the spark schema name was a little different and my find+replace didn't catch that. Re: docs-- I thought I had deleted .gspickle and .msgpack that was originally in there before? Lmk if that doesn't seem the case on your end

image

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-reneeli looks like I mistook the deletion with addition (maths 🤦). Thanks for removing those. Things look good to me!

**Pull request merged**